doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1172,7 +1172,21 @@ Exposing constants to OCaml libraries
    .. seealso:: :ref:`primitive-integers`
 
 .. cmd:: Print Registered
-   :undocumented:
+
+   List the currently registered constants.
+
+.. cmd:: Register @qualid__1 as @qualid__2 for @qualid__3
+
+   Make the constant :n:`@qualid__1` accessible to the "scheme"
+   mechanism for scheme kind :n:`@qualid__2` and inductive
+   :n:`@qualid__3`.
```

Comment:
Move before `Print Registered` so it reads better (first you register, then you can print).

Reply:
I think `Print Registered` is about `Register ... as ...` not `Register ... as ... for...` (for which we have `Print Registered Schemes`) below

Reply:
Yes, schemes are not seen as a subset of regular registrations, they are separate.

Reply:
We could merge the printing commands though

Reply:
also I made a mistake it should be `.. cmd:: Register Scheme ...` not just Register